src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -197,4 +196,20 @@ export class QuestionSubmissionFormComponent implements OnInit {
     return this.feedbackResponseService.isFeedbackResponseDetailsEmpty(
         this.model.questionType, responseDetails);
   }
+
+  /**
+   * Updates validity of all responses in a question.
+   */
+  updateValidity(isValid: boolean): void {
+    const recipientSubmissionForms: FeedbackResponseRecipientSubmissionFormModel[] =
+        this.model.recipientSubmissionForms.slice().map(
+            (model: FeedbackResponseRecipientSubmissionFormModel) => Object.assign({}, model, { isValid }));
+    recipientSubmissionForms.forEach((model: FeedbackResponseRecipientSubmissionFormModel) => {
```

Comment:
So you should remove this line also? It is redundant as L206 already did the job?